js/config.js

```diff
@@ -6,7 +6,7 @@ let respecConfig = {
   specStatus: "WV",
   specType: "PR",
   pubDomain: "logboek",
-  shortName: "logboek-extensie-template",
+  shortName: "logboek-extensie-object",
```

Comment:
Moeten we hier niet `extensie-object` van maken? `logboek` is namelijk al het `pubDomain`